docs/working-with-states/03-listing-states.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Listing states
+weight: 3
+---
+
+Say you have setup the invoice model as follows:
+
+```php
+namespace App;
+
+use App\States\Invoice\Paid;
+use App\States\Invoice\Pending;
+use App\States\Invoice\Declined;
+use App\States\HasStates;
+use App\States\Fulfillment\Partial;
+use App\States\Fulfillment\Complete;
+use App\States\Invoice\InvoiceState;
+use App\States\Fulfillment\Unfulfilled;
+use Illuminate\Database\Eloquent\Model;
+use App\States\Fulfillment\FulfillmentState;
+
+/**
+ * @property \App\States\Invoice\InvoiceState state
+ * @property \App\States\Fulfillment\FulfillmentState fulfillment
+ */
+
```

Comment:
You don't need to list this meta info in this example

Reply:
👍